### PR TITLE
Animation

### LINKED ISSUE #118: Implement loading animation

### PR DESCRIPTION
**Pull request by stacyp2006**
### Participating Group Members:

- Stacy Potten
- Nathan Darrington

### Code Highlights:

- Add the LoadingIcon component to components where we are rendering loading messaging for the user. 
- Remove "Loading..." text from animation because we already have custom messaging where needed. 

### Have Tests Been Added?

- [x] No.
- [ ] Yes, but all tests are not passing.
- [ ] Yes, and all are passing.

### Any background context you want to provide?
- Nathan created this animation previously, this branch just implements it where needed. 

### Issues:

- Addresses #
- Closes #118

### Screenshots (if appropriate):
Screen Shot 2021-01-14 at 2 31 11 PM
\*

### Tracking Consistency:

- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] All new and existing tests passed
- [x] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

**Review by npdarrington — APPROVED** [review]
Thank you for implementing this Stacy!

File: src/components/FindPark.vue [hunk 1]
```diff
@@ -15,6 +15,7 @@
     </router-link>
     <h3 id="location-message" v-if="!this.$store.state.geolocation">
       {{ message }}
+      <h4 v-if="message === 'Retrieving your location...'"><LoadingIcon /></h4>
```
Comment (bretm9) [inline src/components/FindPark.vue:18]: So happy we could get this animation implemented before code fair!

**Review by bretm9 — APPROVED** [review]
Thanks for getting this in. Not many lines edited for this addition, so I don't have much to say other than "Yay!" I'm really glad to see we're utilizing this animation in our app now.

**Review by nicolegooden — APPROVED** [review]
Looks great, thanks Stacy! Glad we could put Nathan's hard work to good use here.

**Pull request merged**